dashboard/app/helpers/levels_helper.rb

```diff
@@ -256,6 +256,9 @@ def app_options
       @app_options[:userSharingDisabled] = current_user.sharing_disabled?
     end
 
+    # TODO: epeach - remove this temporary appOption to indicate the condition in resources tab experiment
+    @app_options[:resourcesTabVisible] = Experiment.enabled?(user: current_user, experiment_name: "instructions-resources-tab")
```

Comment:
Server-side experiments are actually passed through to the client already: https://github.com/code-dot-org/code-dot-org/blob/d9d25b7eed245eebef96c70a687e9f5c46c0c3da/dashboard/app/helpers/levels_helper.rb#L253-L254

If you rename the server-side experiment to `resourcesTab`, your existing `experiments.isEnabled` calls should just work.

I apologize for the total lack of documentation around this, that's my fault.